backend-api/src/functions/services/events/types.ts

```diff
@@ -68,12 +69,23 @@ export const txmaBillingEventNames = [
 
 export type TxmaBillingEventName = (typeof txmaBillingEventNames)[number];
 
+export type EventNames =
+  | GenericEventNames
+  | TxmaBillingEventName
+  | "DCMAW_ASYNC_CLIENT_CREDENTIALS_TOKEN_ISSUED"
+  | "DCMAW_ASYNC_BIOMETRIC_TOKEN_ISSUED";
+
+export type EvidenceWithTxMAContraIndicators = (PassEvidence | FailEvidence) & {
```

Comment:
Suggesting a more generic name since this will also need optional `ciReasons`.

```suggestion
export type VcIssuedEvidence = (PassEvidence | FailEvidence) & {
```

Reply:
Changed it